### PR TITLE
Added recipe for JAGS

### PR DESCRIPTION
**Pull request by micknudsen**

### CODE REVIEW

File: recipes/jags/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  url: https://sourceforge.mirrorservice.org/m/mc/mcmc-jags/JAGS/4.x/Source/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - gcc
```
Comment (dougalsutherland) [inline recipes/jags/meta.yaml]: Do you actually explicitly need the `gcc` package on Linux? Seems unlikely. If it doesn't work with the system clang on Mac, you should probably just do <code>- gcc &nbsp;# [osx]</code> (and similar for `libgcc` in `run`).

File: recipes/jags/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://sourceforge.mirrorservice.org/m/mc/mcmc-jags/JAGS/4.x/Source/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
```
Comment (dougalsutherland) [inline recipes/jags/meta.yaml]: Add the `toolchain` package as a `build` requirement.

File: recipes/jags/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+{% set version = "4.3.0" %}
+{% set sha256 = "8ac5dd57982bfd7d5f0ee384499d62f3e0bb35b5f1660feb368545f1186371fc" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://sourceforge.mirrorservice.org/m/mc/mcmc-jags/JAGS/4.x/Source/{{ name }}-{{ version }}.tar.gz
```
Comment (dougalsutherland) [inline recipes/jags/meta.yaml]: Replacing `4.x` with `{{ version[0] }}.x` might make life very slightly easier when a 5.0 comes out.

**Comment by micknudsen** [comment]
Yes, it fixes the problem. I will go ahead and address the other issues you pointed out.

**Comment by dougalsutherland** [comment]
Ah, I see, it's a Fortran issue. Try using:
```
  build:
    - toolchain
    - gcc             # [osx]
    - libgfortran  # [linux]
    - lapack
  run:
    - libgfortran
    - lapack
```

**Comment by dougalsutherland** [comment]
Okay, I thought we could get away with it but apparently we do need `gcc` as a build dep on Linux too. (I thought we could get away with this setup based on the `scipy` recipe, but apparently not.) Sorry for so much fiddling.

**Comment by micknudsen** [comment]
I can't see why CircleCI is failing. There is just a "spinning wheel" in the failed section. Maybe I should add libgcc back?

**Comment by dougalsutherland** [comment]
You can often convince CircleCI to work by collapsing that output section and then uncollapsing it again. It's dumb.

The failure was:
```
+ jags
/staged-recipes/build_artefacts/jags_1509369825057/_t_env/libexec/jags-terminal: /usr/lib64/libstdc++.so.6: version `GLIBCXX_3.4.15' not found (required by /staged-recipes/build_artefacts/jags_1509369825057/_t_env/libexec/../lib/libjags.so.4)
TESTS FAILED: jags-4.3.0-0.tar.bz2
```
I guess `libgcc` is needed then, maybe. Sorry for all of that. :/

**Comment by micknudsen** [comment]
Does this recipe need more work before it can be merged? I would like to start working on a new recipe, which depends on JAGS.

**Comment by dougalsutherland** [comment]
Nope, looks good. Thanks @micknudsen! https://github.com/conda-forge/jags-feedstock/ will be created soon and the first builds uploaded not long after that.

**Pull request merged**